root/etc/ansible/entrypoint.yml

```diff
@@ -111,37 +111,16 @@
       tags:
         - firstrun
 
-    - name: postgresql db
-      postgresql_db:
-        name="{{ database_name }}"
-        state=present
-        login_host="{{ database_host }}"
-        port={{ database_port }}
-        login_user="{{ database_root_user_postgres }}"
-        login_password="{{ database_root_password_postgres }}"
+    - name: verify PostgreSQL server is available
+      postgresql_ping:
+        login_host: '{{ database_host}}'
+        login_user: '{{ database_user }}'
+        login_password: '{{ database_password }}'
+        db: '{{ database_name }}'
```

Comment:
I think you forget to specify the port of the pgsql server.
Consider using `port: '{{ database_port }}'`